queue_services/entity-emailer/src/entity_emailer/email_processors/special_resolution_helper.py

```diff
@@ -59,11 +59,13 @@ def get_completed_pdfs(
     # Change of Name
     if name_changed:
         name_change = requests.get(
-            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
-            '?type=changeOfName',
-            headers=headers
-        )
-        if name_change.status_code == HTTPStatus.OK:
+                f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
+                '?type=certificateOfNameChange',
+                headers=headers
+            )
+        if name_change.status_code != HTTPStatus.OK:
+            logger.error('Failed to get certificateOfNameChange pdf for filing: %s', filing.id)
+        else:
```

Comment:
You'll want to check here for httpStatus.OK, otherwise other codes could slip in here.